exporter/datadogexporter/config.go

```diff
@@ -117,7 +117,7 @@ type HistogramConfig struct {
 
 	// SendCountSum states if the export should send .sum and .count metrics for histograms.
 	// The default is false.
-	// Deprecated: [v0.75.0] Use `send_aggregation_metrics` (HistogramConfig.SendAggregations) instead.
+	// Deprecated: [v0.76.2] Use `send_aggregation_metrics` (HistogramConfig.SendAggregations) instead.
```

Comment:
This shouldnt be changed

Reply:
Fixed the two you mentioned and another.

Reply:
Its possible github is losing its mind with this large of a changelist, but im still seeing 76.2 here